index.html

```diff
@@ -5,175 +5,167 @@
+                        forms and gestural marks to achieve its effect
+                        Abstract art is art that does
+                        not attempt to represent an accurate depiction of a visual reality but instead use shapes,
+                        colours,
+                        forms and gestural marks to achieve its effect
+                    </P>
+                    <h4>2022.10.18(FRI) ~19(SAT)</h4>
+                    <h5>“Bringing a voice to our young generation”</h5>
+                </div>
+            </div>
         </section>
-            <section class="card">
-                <div class="card-items">
-                        <h3 class="heading-three-cards">Main Program</h3>
-                    <div class="card-grid">
-                        <div class="card-item-child">
-                            <div><span><i class=" fad fa-solid fa-person-chalkboard"></i></span></div>
-                            <h3 class="lecture">lecture</h3>
-                            <p class="card-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
-                                 voluptatum dolore.</p>
-
-                        </div>
-                        <div class="card-item-child">
-                            <div><span><i class="fad fa-solid fa-person-chalkboard"></i></span></div>
-                            <h3 class="lecture">lecture</h3>
-                            <p class="card-paragraph" >Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
-                                 voluptatum dolore.</p>
-
-                        </div>
-                        <div class="card-item-child">
-                            <div><span><i class="fad fa-solid fa-person-chalkboard"></i></span></div>
-                            <h3 class="lecture">lecture</h3>
-                            <p class="card-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
-                                 voluptatum dolore.</p>
-
-                        </div>
-                        <div class="card-item-child">
-                            <div><span><i class="fad fa-solid fa-person-chalkboard"></i></span></div>
-                            <h3 class="lecture">lecture</h3>
-                            <p class="card-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
-                                 voluptatum dolore.</p>
-
-                        </div>
-                        <div class="card-item-child">
-                            <div><span><i class="fad fa-solid fa-person-chalkboard "></i></span></div>
-                            <h3 class="lecture">lecture</h3>
-                            <p class="card-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
-                                 voluptatum dolore.</p>
-
-                        </div>
-                    </div>
```

Comment:
- [ ] On the main program section the margin on the left is not the same as the margin on the right. Please make sure that the margins on both sides are the same.

Reply:
I already did that as well